.npmignore

```diff
@@ -0,0 +1,18 @@
+*.log
+*.pid
+*.seed
+*.swp
+*.un~
+*~
+.DS_Store
+.netrwhist
+.node_repl_history
+.npm
+CONTRIBUTING*
```

Comment:
when installed as a `node_module`, most folks don't look at `.md` files - and that I copy and pasted this from somewhere. good call though; I agree it's a good idea to keep.